packages/store/ts/config/v2/input.ts

```diff
@@ -26,7 +26,7 @@ export type TablesInput = {
 
 export type StoreInput = {
   readonly namespace?: string;
-  readonly tables: TablesInput;
+  readonly tables?: TablesInput;
```

Comment:
is it necessary for this to be optional?

Reply:
this was an oversight, you don't have to provide tables in your config

Reply:
when would you want to specify a config without tables?